portworx/templates/portworx-storagecluster.yaml

```diff
@@ -26,7 +26,7 @@ spec:
     {{- end }}
     # TODO: Change this hardcoded image path to an ECR registry path with px-enterprise image (PWX-27961)
     - name: PX_IMAGE
-      value: docker.io/portworx/px-enterprise:2.12.0
+      value: 709825985650.dkr.ecr.us-east-1.amazonaws.com/portworx/px-enterprise:pradyuman_2.12.1
```

Comment:
How come there is a personal name in the change ? Is it intended ?

Reply:
In marketplace ECR registry, one cannot push images with the same tag twice with the intention to update it.
So once i push something with the tag for example 2.12.1, I won't be able to update it later if there are any changes
The official images for px-enterprise 2.12.1 was not out then, so I created a tag with my name for my development purpose. I will update it once the official images is out